### PR TITLE
get the existing mask to modify that more

### PR DESCRIPTION
**Pull request by sameera2004**

### CODE REVIEW

File: xray_vision/mask/manual_mask.py [hunk 1]
```diff
@@ -56,9 +56,9 @@
 
 class ManualMask(object):
     @ensure_ax_meth
-    def __init__(self, ax, image, cmap='gray',
```
Comment (tacaswell) [inline xray_vision/mask/manual_mask.py]: new kwargs should go at the end to mantain API.

Comment (sameera2004) [inline xray_vision/mask/manual_mask.py]: Sure, I will put that at the end

**Comment by sameera2004** [comment]
@tacaswell I added mask to the end, ready to review.

**Comment by tacaswell** [comment]
Can you also rebase this onto current master?  There are some merge commits in here.

**Comment by tacaswell** [comment]
other than that :+1:

**Comment by sameera2004** [comment]
@tacawell it is ready for review

**Comment by sameera2004** [comment]
Thanks @tacaswell

**Pull request merged**